pico/configdriven/runtime/src/main/java/io/helidon/pico/configdriven/runtime/DefaultConfigBeanBuilderValidatorProvider.java

```diff
@@ -20,12 +20,11 @@
 import io.helidon.builder.config.spi.ConfigBeanBuilderValidatorProvider;
 import io.helidon.common.LazyValue;
 import io.helidon.common.Weight;
-import io.helidon.common.Weighted;
 
 /**
  * Service-loaded provider for {@link io.helidon.builder.config.spi.ConfigResolverProvider}.
  */
-@Weight(Weighted.DEFAULT_WEIGHT)
+@Weight(io.helidon.pico.api.ServiceInfoBasics.DEFAULT_PICO_WEIGHT)
```

Comment:
Why is it fully qualified? There should not be a naming conflict.

Reply:
Seems to be in all usages

Reply:
Yes, it was just easier to do it that way. Is that somehow a problem?  I did the same for the javadoc.

Reply:
pushed